From: Michal Marek <mmarek@suse.cz>
Subject: kabi: Restore include in arch/s390/kernel/setup.c
Patch-mainline: never

Removing the include in patches.arch/s390-sles12-13-11-ftrace.patch changed
the checksum of the 'vmemmap' and 'lowcore_ptr' symbols needlessly.

Signed-off-by: Michal Marek <mmarek@suse.cz>

---
 arch/s390/kernel/setup.c |    1 +
 1 file changed, 1 insertion(+)

--- linux-3.12-SLE12.orig/arch/s390/kernel/setup.c
+++ linux-3.12-SLE12/arch/s390/kernel/setup.c
@@ -40,6 +40,7 @@
 #include <linux/ctype.h>
 #include <linux/reboot.h>
 #include <linux/topology.h>
+#include <linux/ftrace.h>
 #include <linux/kexec.h>
 #include <linux/crash_dump.h>
 #include <linux/memory.h>
